just/bluefin-system.just

```diff
@@ -126,6 +126,7 @@ ptyxis-transparency opacity="0.95":
 
 # Configure docker,incus-admin,lxd,libvirt container manager permissions
 dx-group:
+    cat /usr/lib/group | grep 'docker:\|incus-admin:\|lxd:\|libvirt:' | sudo tee -a /etc/group >/dev/null
```

Comment:
Please check if the group exists in /etc/group first. This appends them everytime.